src/test_utils/katana/mod.rs

```diff
@@ -35,16 +35,17 @@ fn load_genesis() -> Genesis {
 /// If `with_dumped_state` is true, the config will be initialized with the dumped state.
 pub fn katana_config() -> StarknetConfig {
     let max_steps = std::u32::MAX;
+    let mut genesis = load_genesis();
+    genesis.gas_prices = GasPrices { eth: 1, strk: 0 };
```

Comment:
I think we were wrong to have positive GasPrices here, cc @greged93 
we can delete this line and keep the GasPrices as a default null value

Reply:
i added this because just below this we where setting this `gas_price`, and now with was moved inside `gensis` to had to manually update it after loading genesis. 

should i remove this?